### ISSUE TITLE
KAP140 display disappearing

### ISSUE THREAD

**Issue by dsatgh** [HHS81/c182s#593]
When setting rendering mode to low specifications, the KAP140 display disappears depending on the viewing angle. I know it's the glass in the ac file that's causing it, but removing it is not a nice solution, if there's a better one. Maybe this should better be solved, if possible, in FG, SG?

**Comment by hbeni** [comment]
I did not find a better solution than to hide the glass at the 182.

But probably its better to file an issue with the kap140 upstream project: https://github.com/reiszner/KAP140/issues
Or is it already the same as already reported there: https://github.com/reiszner/KAP140/issues/16

**Comment by hbeni** [comment]
Reading the ticket; yes its an effects issue and needs to be fixed in flightgear itself.

**Comment by TheFGFSEagle** [comment]
Is model-transparent applied to it ? 'Cause I didn't notice it on the C310A.

**Comment by dsatgh** [comment]
Has your C310A a KAP140? I couldn't find it. But on the other instruments so far I've not seen parts disappearing behind the glass.

**Comment by TheFGFSEagle** [comment]
No it doesn't have a KAP140 but other displays, and those are not disappearing.

**Comment by dsatgh** [comment]
I tested with model-transparent and it made no difference. However the best solution I discovered is this:
```
  <animation>
    <type>select</type>
    <condition>
      <equals>
        <property>sim/rendering/shaders/skydome</property>
        <value>true</value>
      </equals>
    </condition>
    <object-name>glass</object-name>
  </animation>
```

**Comment by wlbragg** [comment]
Does it also disappear under low settings in the c172p?

**Comment by dsatgh** [comment]
Yes the c172p has the same problem. I would say even worse with more instruments not showing.

**Comment by wlbragg** [comment]
Yeah, it looks like it is just the TC and the 2 VOR's. I don't recall this ever doing this in the past. Is this something relatively new? Some change to the core that caused it? I would have absolutely noticed this when I was working on the effects for hi res VS lo res cpu/gpu specs.

**Comment by dsatgh** [comment]
As I'm relatively new to FG after years without due to very low spec hardware, I don't know when it changed. IMHO it is all but impossible to develop for all combinations of hardware, software and FG settings. So not worth spending time for a use case with me probably being the only one  :)
BTW I can run more advanced settings with my hardware nowadays.

**Comment by dsatgh** [comment]
Problem solved with FG 2024.1.1 ;)
